[CI] LogsdbIndexingRollingUpgradeIT testIndexing {upgradedNodes=1} failing

**Build Scans:**
- [elasticsearch-periodic #4628 / openjdk21_8.17.0_java-fips-matrix-bwc](https://gradle-enterprise.elastic.co/s/towpqkiv27o62)
- [elasticsearch-periodic #4618 / openjdk21_8.17.0_java-fips-matrix-bwc](https://gradle-enterprise.elastic.co/s/v77kxiiic2tky)

**Reproduction Line:**
```
./gradlew ":qa:rolling-upgrade:v8.17.0#bwcTest" -Dtests.class="org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT" -Dtests.method="testIndexing {upgradedNodes=0}" -Dtests.seed=1515255A85317F7A -Dtests.bwc=true -Dtests.locale=en-BM -Dtests.timezone=EST5EDT -Druntime.java=21 -Dtests.fips.enabled=true
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.upgrades.LogsdbIndexingRollingUpgradeIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testIndexing%20%7BupgradedNodes%3D1%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: "Trial was already activated."
     but: was "Operation failed: Trial was already activated."
```

**Issue Reasons:**
- [main] 2 failures in test testIndexing {upgradedNodes=1} (1.0% fail rate in 209 executions)
- [main] 2 failures in step openjdk21_8.17.0_java-fips-matrix-bwc (66.7% fail rate in 3 executions)
- [main] 2 failures in pipeline elasticsearch-periodic (66.7% fail rate in 3 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/LogsdbIndexingRollingUpgradeIT.java
+++ b/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/LogsdbIndexingRollingUpgradeIT.java
@@ -28,10 +28,7 @@ import java.util.Map;
 import static org.elasticsearch.upgrades.LogsIndexModeRollingUpgradeIT.enableLogsdbByDefault;
 import static org.elasticsearch.upgrades.LogsIndexModeRollingUpgradeIT.getWriteBackingIndex;
 import static org.elasticsearch.upgrades.TsdbIT.formatInstant;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.greaterThan;
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
-import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.*;
 
 public class LogsdbIndexingRollingUpgradeIT extends AbstractRollingUpgradeTestCase {
 
@@ -240,7 +237,7 @@ public class LogsdbIndexingRollingUpgradeIT extends AbstractRollingUpgradeTestCa
         } catch (ResponseException e) {
             var responseBody = entityAsMap(e.getResponse());
             String error = ObjectPath.evaluate(responseBody, "error_message");
-            assertThat(error, equalTo("Trial was already activated."));
+            assertThat(error, containsString("Trial was already activated."));
         }
     }
 

